cli/slashingExport_test.go

```diff
@@ -0,0 +1,187 @@
+			},
+		},
+		{
+			name: "stop-validator flag",
+			args: []string{"prysm", "--stop-validator"},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "prysm",
+				Network:         "mainnet",
+				StopValidator:   true,
+				StartValidator:  false,
+				GenerationPath:  configs.DefaultDockerComposeScriptsPath,
+				Out:             filepath.Join(configs.DefaultDockerComposeScriptsPath, "slashing_export.json"),
+			},
+		},
+		{
+			name: "start-validator flag",
+			args: []string{"teku", "--start-validator"},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "teku",
+				Network:         "mainnet",
+				StopValidator:   false,
+				StartValidator:  true,
+				GenerationPath:  configs.DefaultDockerComposeScriptsPath,
+				Out:             filepath.Join(configs.DefaultDockerComposeScriptsPath, "slashing_export.json"),
+			},
+		},
+		{
+			name: "path flag",
+			args: []string{"teku", "--path", filepath.Join("custom", "dir")},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "teku",
+				Network:         "mainnet",
+				StopValidator:   false,
+				StartValidator:  false,
+				GenerationPath:  filepath.Join("custom", "dir"),
+				Out:             filepath.Join("custom", "dir", "slashing_export.json"),
+			},
+		},
+		{
+			name: "path shorthand flag",
+			args: []string{"teku", "-p", filepath.Join("custom", "dir")},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "teku",
+				Network:         "mainnet",
+				StopValidator:   false,
+				StartValidator:  false,
+				GenerationPath:  filepath.Join("custom", "dir"),
+				Out:             filepath.Join("custom", "dir", "slashing_export.json"),
+			},
+		},
+		{
+			name: "out flag",
+			args: []string{"lodestar", "--out", filepath.Join("custom", "out", "file.json")},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "lodestar",
+				Network:         "mainnet",
+				StopValidator:   false,
+				StartValidator:  false,
+				GenerationPath:  configs.DefaultDockerComposeScriptsPath,
+				Out:             filepath.Join("custom", "out", "file.json"),
+			},
+		},
+		{
+			name: "out shorthand flag",
+			args: []string{"lodestar", "-o", filepath.Join("custom", "out", "file.json")},
+			actionOptions: actions.SlashingExportOptions{
+				ValidatorClient: "lodestar",
+				Network:         "mainnet",
+				StopValidator:   false,
+				StartValidator:  false,
+				GenerationPath:  configs.DefaultDockerComposeScriptsPath,
+				Out:             filepath.Join("custom", "out", "file.json"),
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			defer ctrl.Finish()
+
+			actions := mock_actions.NewMockSedgeActions(ctrl)
+			actions.EXPECT().ExportSlashingInterchangeData(tt.actionOptions).Times(1)
+
+			slashingImportCmd := cli.SlashingExportCmd(actions)
+			slashingImportCmd.SetArgs(tt.args)
+			err := slashingImportCmd.Execute()
```

Comment:
Copy-pasted detected again

Reply:
I wonder how this test is passing bc you are mocking the call of `ExportSlashingInterchangeData` instead of `ImportSlashingInterchangeData`. Make sure first that this test should fail bc `ExportSlashingInterchangeData` should be called once and is not being called.

Reply:
Please read [this explanation](https://github.com/NethermindEth/sedge/pull/209#discussion_r1071588289). Fixed here https://github.com/NethermindEth/sedge/pull/209/commits/94ad27c973701965d682e605df3e6dc263812b7a

Reply:
You are right